src/spatial/2d.js

```diff
@@ -23,6 +23,19 @@ Crafty.extend({
         if (width > 0)
             return new Array(width + (/\./.test(number) ? 2 : 1)).join('0') + number;
         return number.toString();
+    },
+    /**@
+     * #Crafty.intersect
+     * @category 2D
+     * Utility method to determine if two rectangles intersect.
+     * @sign public Boolean Crafty.intersect(Object rectA, Object rectA)
+     * @param rectA - An object that must have the `_x, _y, _w, _h` values as properties
+     * @param rectB - An object that must have the `_x, _y, _w, _h` values as properties
+     * @return true if the rectangles intersect; false otherwise
+     */
+    intersect: function (rectA, rectB) {
```

Comment:
Is there actually a reason for this right now?  (i.e., is there any reason to have both this *and* rectManager.intersect?)